app/management/tracker_client.py

```diff
@@ -503,6 +542,7 @@ def save_model(
         model_name: str,
         model_manager: ModelManager,
         validation_status: str = "pending",
+        model_type: Optional[str] = None,
```

Comment:
All CMS models have the ModelType, hence there's no need to make the argument optional and mlflow.set_tag() will not set a None value.